install/kubernetes/cilium/charts/agent/templates/servicemonitor.yaml

```diff
@@ -21,3 +22,26 @@ spec:
     honorLabels: true
     path: /metrics
 {{- end }}
+{{- if .Values.global.hubble.hubbleUi.enabled }}
```

Comment:
This if statement can be removed, the service monitor is not dependent on Hubble UI.

Reply:
Addressed review comments.